src/rust/ide/lib/analytics/src/remote_log.rs

```diff
@@ -9,27 +9,44 @@ pub use wasm_bindgen::prelude::*;
 
 
 #[wasm_bindgen(inline_js="
-export function _remote_log(msg, data) {
-    if (window !== undefined && window.enso !== undefined && window.enso.remote_log !== undefined) {
-        window.enso.remote_log(msg,{data:data})
+export function _remote_log(msg, value) {
+    if (window !== undefined && window.enso !== undefined && window.enso.remoteLog !== undefined) {
+        try {
+            if (value === undefined && value !== null) {
+                window.enso.remoteLog(msg)
+            } else {
+                window.enso.remoteLog(msg,value)
+            }
```

Comment:
Wait, this can be simplified a lot - just use `window.enso.remoteLog(msg,value)` - `value` can be undefined, you don't need to check it!